src/middlewared/middlewared/plugins/system.py

```diff
@@ -962,6 +973,19 @@ def _get_index(country_columns, column):
 
         When `syslogserver` is defined, `sysloglevel` makes sure that logs matching the specified level are sent.
```

Comment:
Could you also please put a note in the docstring here about the deprecation?
Some along the lines these attributes are deprecated as of 11.3 and will be permanently moved for 12?

Reply:
It shows up in http://freenas_ip/api/docs/